server/runtime/src/main/server/README.adoc

```diff
@@ -42,6 +42,14 @@ Server configuration extends ${infinispan.brand.name} configuration with the fol
 `socket-bindings`:: Maps endpoint connectors to interfaces and ports.
 
 The default configuration file is `<SERVER_HOME>/server/conf/infinispan.xml`.
+${infinispan.brand.name} provides other ready-to-use configuration files that are primarily for development and testing purposes.
+
+`<SERVER_HOME>/server/conf/` provides the following configuration files:
+
+`infinispan-dev-mode.xml`:: Configures ${infinispan.brand.name} Server specifically for cross-site replication with IP multicast discovery. The configuration provides `BASIC` authentication to connect to the Hot Rod and REST endpoints.
```

Comment:
I would add explicitly that this is for dev mode only, not for production in any case

Reply:
Updated. Better be safe than sorry